admission-webhook/webhook/base/kustomization.yaml

```diff
@@ -1,33 +1,36 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
```

Comment:
you could probably add a 
```
namePrefix: admission-webhook-
```

then in all your other resources like ServiceAccount name would be shortened
eg 
admission-webhook-bootstrap-role-binding -> bootstrap-role-binding

some names would now need a suffix like deployment 
eg
deployment -> admission-webhook-deployment

I did have some problems getting the nameprefix to apply to ClusterRole, ClusterRolebinding in tf-job-operator. But the work around there would be to leave their names with the admission-webhook- part as is.

Reply:
will do.

Reply:
The script "create_ca.sh" in the package uses the name of webhook service (currently hardcoded). I can make it as an argument, however, have not found any good solution to pass namePrefix to the resources.  I can make a json patch to parametrize the webhook service name  but then user needs to change both name Prefix and the json file. Any idea?  @kkasravi